sdk/data/azcosmos/cosmos_client_options.go

```diff
@@ -15,4 +17,11 @@ type ClientOptions struct {
 	EnableContentResponseOnWrite bool
 	// PreferredRegions is a list of regions to be used when initializing the client in case the default region fails.
 	PreferredRegions []string
+	// DisableCrossRegionalRetries is a boolean flag letting the SDK know whether or not it should apply cross-regional retries to requests.
```

Comment:
Add another `//` comment line below with `For more information, see https://learn.microsoft.com/azure/cosmos-db/nosql/troubleshoot-sdk-availability

Reply:
as per our SDK conversation I'll be removing this entirely